tools/checkpoint_conversion/convert_xlm_roberta_checkpoints.py

```diff
@@ -0,0 +1,334 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+import numpy as np
+import tensorflow as tf
+import torch
+import transformers
+from absl import app
+from absl import flags
+from checkpoint_conversion_utils import get_md5_checksum
```

Comment:
another thing I just noticed in this PR (but probably applies to others), we try to do our imports from the base of the repo, and keep it below in the local import block. can we do the same here?

see:
https://github.com/keras-team/keras-nlp/blob/master/examples/bert/bert_preprocess.py#L48

Ideally all these tools should be runnable from the repo base too. `python tools/checkpoint_conversion/...`